src/Telegram.Bot/Enums/Gender.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ReSharper disable once CheckNamespace
+// ReSharper disable InconsistentNaming
+namespace Telegram.Bot
+{
+    public static partial class Enums
+    {
+        /// <summary>
+        /// Gender, male or female
+        /// </summary>
+        public static class Gender
+        {
+            /// <summary>
+            /// Male
+            /// </summary>
+            public const string male = nameof(male);
```

Comment:
Are you sure about that? Public static fields should be in pascal case according to the MS [style guide](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-fields).

Reply:
I was looking for readability but OK, I change it.